### PR TITLE
Cleaned ContentModule some and "Errors" were changed to "Warn" due to the nature of the exceptions they were logging for.

### PR DESCRIPTION
**Pull request by dlamkins**

### CODE REVIEW

File: Blish HUD/GameServices/ContentService.cs [hunk 1]
```diff
@@ -132,7 +132,7 @@ public enum FontStyle {
 
         private static Texture2D TextureFromFileSystem(string filepath) {
             if (!File.Exists(REF_FILE)) {
-                Console.WriteLine($"{REF_FILE} is missing!  Lots of assets will be missing!");
+                Logger.Warn("{refFileName} is missing!  Lots of assets will be missing!", REF_FILE);
```
Comment (greaka) [inline Blish HUD/GameServices/ContentService.cs]: The usage of `refFileName` is confusing. Either stick to `{}` with the format string format, or use the string interpolation `$"{REF_FILE}"` please.

**Review by greaka — CHANGES_REQUESTED** [review]
looks good so far
Since you didnt open a draft pr, I assume that you want to merge this and reviewed it.

File: Blish HUD/GameServices/ContentService.cs (hunk 1 above)
Comment (dlamkins) [reply Blish HUD/GameServices/ContentService.cs]: Could you please elaborate on that first option?  NLog best practice is to allow for the additional params to be sent to the target logger.  https://github.com/NLog/NLog/wiki/How-to-use-structured-logging

Comment (greaka) [reply Blish HUD/GameServices/ContentService.cs]: Ah, didn't know of that! I didn't use NLog in a *long* time, so I missed the structured logging update. I have to read into that.

**Review by greaka — APPROVED** [review]

**Review by greaka — APPROVED** [review]

**Comment by greaka** [comment]
hmm, github seems to not like my late approvals..

**Comment by dlamkins** [comment]
> hmm, github seems to not like my late approvals..
> EDIT: found the problem, I have no write access anymore 😄

All fixed!

**Pull request merged**